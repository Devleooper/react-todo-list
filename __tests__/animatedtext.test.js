import React from "react";
import renderer from "react-test-renderer";
import AnimatedText from "./../src/components/animated/text/animated.text";

it("should render correctly", () => {
  const tree = renderer.create(<AnimatedText />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("should render correctly with props", () => {
  const textStyle = {
    color: "#282c34",
    fontSize: "36pt",
    textAlign: "center",
  };
  const tree = renderer
    .create(<AnimatedText style={textStyle} text="+1" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
