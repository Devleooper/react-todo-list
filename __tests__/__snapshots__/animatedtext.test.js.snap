// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <h1 />
</div>
`;

exports[`should render correctly with props 1`] = `
<div
  style={
    Object {
      "color": "#282c34",
      "fontSize": "36pt",
      "textAlign": "center",
    }
  }
>
  <h1>
    +1
  </h1>
</div>
`;
